### ISSUE TITLE
net: TestInterfaces failing on windows

### ISSUE THREAD

**Issue by bradfitz** [golang/go#10214]
I rebuilt the Windows image (the same as normal, following the checklist in x/build/env), but now TestInterfaces is failing for some reason:

```
--- FAIL: TestInterfaces (0.01s)
	interface_test.go:71: "Local Area Connection": flags "up|broadcast|multicast", ifindex 12, mtu 1430
	interface_test.go:72: hardware address "42:01:0a:f0:24:ca:00:00"
	interface_test.go:169: interface address "10.240.36.202/16"
	interface_test.go:190: joined group address "224.0.0.1"
	interface_test.go:190: joined group address "224.0.0.252"
	interface_test.go:90: got 0; want more than or equal to one
	interface_test.go:102: got 0; want more than or equal to one
FAIL
FAIL	net	4.015s
```

e.g. http://build.golang.org/log/abdd7adc73e7a65beec1773f5a04b3ed7065d3a6

/cc @alexbrainman @mikioh

**Comment by bradfitz** [comment]
I'll also note that this is a bad error message:

```
got 0; want more than or equal to one
```

Zero whats?

**Comment by alexbrainman** [comment]
I will see what the problem is tomorrow. Thank you for getting builders up again @bradfitz .

Alex

**Comment by bradfitz** [comment]
From a quick read, it seems that the Windows builder appears to pass the `supportsIPv6` test, and has one non-loopback interface ("Local Area Connection") but no interface (0 out of 1) has an IPv6 address.

From https://cloud.google.com/compute/docs/networking#networks :

> All Compute Engine networks use the IPv4 protocol. Compute Engine currently does not support IPv6. However, Google is a major advocate of IPv6 and it is an important future direction.

I'm not sure what `TestInterfaces` is trying to test.  The `probeIPv6Stack` function just tries to make & bind some syscall.AF_INET6 sockets, which works.  So I guess the assumption is that there should be some interface with an IPv6 address to which they must be bound?

I'll also note that when I rebuilt this Windows image, Google has in the meantime also updated the base layer Windows image (it's now 30GB instead of 100GB, and has some new GCE-specific tweaks I think), so it's not identical to what it was before, even though I followed the same setup procedures.

To be clear, I'm not working on this bug. Help wanted.
